recipe/meta.yaml

```diff
@@ -22,16 +22,16 @@ requirements:
     - meson-python                           # [build_platform != target_platform]
     - ninja                                  # [build_platform != target_platform]
     - cmake
+    - pkg-config
     - {{ compiler('c') }}
     - {{ compiler('fortran') }}
   host:
     - python
     - pip
     - setuptools
     - meson-python >=0.14
-    - setuptools_scm >=8
-    - pkg-config 
-    - jupyter
+    - setuptools_scm >=8 
+    - jupyter_core
```

Comment:
Please ensure that this change, `jupyter` -> `jupyter_core` didn't accidentally delete a feature of your package.

I see that `jupyter` is used in some tests.
https://github.com/SHTOOLS/SHTOOLS/blob/77ecaabbe90aee89ecc252317c8c6b4258e59bad/examples/notebooks/meson.build#L4

We are honestly pretty "light on tests" at conda-forge to favor ease of building.

Maybe you can just remove `jupyter` from a build time dependency altogether?